### PR TITLE
Prep for release of adls stg95 preview

### PR DESCRIPTION
**Pull request by tanyasethi-msft**
This PR also corrects the shared key algorithm for adls, and a few linter errors for the pipeline to pass

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to module CHANGELOG.md are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
https://github.com/Azure/azure-sdk-for-go/pull/23536 updated the latest two versions of Go and it appears that the code coverage numbers are calculated differently now. It appears that previously, the percentages in the internal content didn't contribute to the overall coverage percentage, and now they do.

**Comment by tanyasethi-msft** [comment]
The azdatalake coverage numbers are changed temporarily until we have a fix for the issue stated in the above comment.

File: eng/config.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
         },
         {
             "Name": "azdatalake",
-            "CoverageGoal": 0.60
+            "CoverageGoal": 0.00
```
Comment (jhendrixMSFT) [inline eng/config.json:41]: According to the test results, current CC is 26.17%. Maybe set this to something like 0.25?

Comment (tanyasethi-msft) [reply eng/config.json]: Fair, changed to 0.25 for now

**Pull request merged**